src/plugins/controls/public/options_list/types.ts

```diff
@@ -29,3 +27,5 @@ export type OptionsListReduxState = ReduxEmbeddableState<
   ControlOutput,
   OptionsListComponentState
 >;
+
+export * from '../../common/options_list/types';
```

Comment:
I was making it match the `range_slider` types file, which also has it at the bottom. But I could move both to the top instead? Don't really have a strong preference - what are your thoughts?

Reply:
Hmmmmm I guess you can leave it at the bottom. It's an export so it's not like we need to do all of them at the top. We have a clear pattern for imports, but I don't think we've discussed exports. I don't know that we need to organize exports in any particular way besides just having them below imports? What do you think @ThomThomson

Reply:
Where are these exports being used? Can we not just import from common in those places rather than _reexporting_ here? I think it's usually best not to re-export if it can be avoided, because it can make things more difficult to follow.

Reply:
Very fair point - the `common` export actually seems to be used more frequently than the re-exported `public` types (at least for options list - haven't checked range slider yet), so I think it makes the most sense to just avoid re-exporting stuff even if it results in shorter paths 👍 I'll make the change to this PR since it should be only a few changes